Cargo.toml

```diff
@@ -6,9 +6,9 @@ members = [
    "derive",
    "lib",
    "indexer",
-   "indexer-samples/*",
    "schema"
 ]
+exclude = ["examples"]
```

Comment:
Could you elaborate a little on why we want to remove these from the main workspace? It would be nice to share a workspace if possible in order to share build artifacts, otherwise every example will require it's own target directory and building the entire set of `fuel-indexer` deps each time.

Reply:
@mitchmindtree So yea,
- When I try to `cargo check` in an example directory, without the examples exclusion, I get the `current package believes it's in a workspace when it's not` error
- Further the `Run Cargo.toml linter` CI check fails because there's no `Cargo.toml` in the `examples/` directory root (if the exclusion is not included)
- I noticed that we also do this (exclude examples) [in the sway repo](https://github.com/FuelLabs/sway/blob/master/Cargo.toml#L25) so I thought it made since to exclude here as well
- I also figured we'd build these example projects on CI later so that we can ensure they're functional/workable separately from the fuel-indexer project, but let me know if you think we should be including these examples

Reply:
- Ah, now seeing that `examples` in the `workspace` does not work. But `examples/*` does 👌 
- However, given the state of the project (still a WIP) I would want to see what you think about temporarily keeping `examples/*` out of the `workspace` until https://github.com/FuelLabs/fuel-indexer/pull/32 is merged?
  - `cargo` complains a bit about the examples, as is

Reply:
> cargo complains a bit about the examples, as is

Can you post the errors? Or remove this exclude line and re-add `examples/*` to the members list so we can see them show up in CI? That way we might be able to help resolve whatever errors are occurring.

I'd prefer not to exclude the examples if we can avoid it as it will result in a lot of `Cargo.lock` noise and greatly increase the CI build times.

Reply:
@mitchmindtree Sounds good. Feel free to check the CI errors